libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -121,10 +124,24 @@ bool AP_RangeFinder_LightWareSerial::get_reading(float &reading_m)
 
     // all readings were invalid so return out-of-range-high value
     if (invalid_count > 0) {
-        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, max_distance_cm() + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
+        reading_m = MIN(MAX(LIGHTWARE_DIST_MAX_CM, distance_cm_max + LIGHTWARE_OUT_OF_RANGE_ADD_CM), UINT16_MAX) * 0.01f;
         return true;
     }
 
     // no readings so return false
     return false;
 }
+
+// check to see if distance returned by the LiDAR is a known lost-signal distance flag
+bool AP_RangeFinder_LightWareSerial::is_lost_signal_distance(float distance_cm, int16_t distance_cm_max)
```

Comment:
Let's keep this in the integer domain rather than moving into float.  Way fewer bytes :-)
```suggestion
bool AP_RangeFinder_LightWareSerial::is_lost_signal_distance(uint16_t distance_cm, int16_t distance_cm_max)
```